src/ontodia/diagram/model.ts

```diff
@@ -41,9 +41,10 @@ export class DiagramModel extends Backbone.Model {
     elements: { [id: string]: Element } = {};
     linksByType: { [type: string]: Link[] } = {};
 
-    constructor(isViewOnly = false) {
+    constructor(isViewOnly = false, linkStyles?: {link?: Object; label?: Object; router?: string; connector?: string}) {
         super();
         this.set('isViewOnly', isViewOnly);
+        this.set('linkStyles', linkStyles);
```

Comment:
It is a bad idea to store `linkStyles` inside diagram **model**, it should be part of diagram **view**.
Maybe there is something I don't see that requires to use model?